tests/templateexample/conf.py

```diff
@@ -21,4 +21,4 @@
 autoapi_file_pattern = "*.py"
 autoapi_template_dir = "template_overrides"
 
-exclude_patterns = [autoapi_template_dir]
+exclude_patterns += [autoapi_template_dir]
```

Comment:
`exclude_patterns` is declared further ahead with `exclude_patterns = ["_build"]`, but re-declared. This change instead appends to the list.